app/services/sendgrid_handler.rb

```diff
@@ -1,32 +1,77 @@
 # frozen_string_literal: true
 
 class SendgridHandler < ServiceHandlerBase
-  def self.link_sendgrid_msg_to_message(message_id, sendgrid_msg_id)
-    sendgrid_msg = SendgridMsg.find_by(id: sendgrid_msg_id)
-    if sendgrid_msg.nil?
-      log_error("Unable to find sendgrid_msg (#{sendgrid_msg_id}). Did not link message (#{message_id})")
-      return false
-    end
-    message = Message.find_by(id: message_id)
-    if message.nil?
-      log_error("Unable to find message (#{message_id}). Did not link sendgrid_msg (#{sendgrid_msg_id})")
-      return false
-    end
-    message.sendgrid_msg_id = sendgrid_msg_id
+  def self.send(message_vo)
+    template_data = message_vo.email_message
+    template_data['email_subject'] = message_vo.email_subject
+
+    sendgrid_vo = create_sendgrid_msg(message_vo, template_data)
+
+    SendgridHandler.send_msg(sendgrid_vo) # <= Uncomment to test synchronously
```

Comment:
why is this uncommented and why are these always running different code for sync vs async?

Reply:
Every time I push this for a PR, I re-comment this line. The "synchronous" methods are there b/c it helps _me) interactively test the flow. I will remove this line (and uncomment the async line) before the code goes live.